docs/technical-doc/api.rst

```diff
@@ -478,11 +489,9 @@ pool.
                 "gas_price": Number, // Amount
             }
         },
-        "sender_public_key": String
-        },
-        "signature": String
     }
-    ]]
+
+For the signature you need to use the bytes of the public_key and content in byte compact concatenated and sign it with blake3.
```

Comment:
Blake3 is a hashing algorithm, not a signature algorithm. cc @AurelienFT

Reply:
Ah yes it's schnorr my bad. Will change it

Reply:
Done in #2744